blazor/ClientSideExample/ClientSideExample/Shared/NavMenu.razor

```diff
@@ -0,0 +1,37 @@
+﻿<div class="top-row pl-4 navbar navbar-dark">
+    <a class="navbar-brand" href="">ClientSideExample</a>
+    <button class="navbar-toggler" @onclick="ToggleNavMenu">
+        <span class="navbar-toggler-icon"></span>
+    </button>
+</div>
+
+<div class="@NavMenuCssClass" @onclick="ToggleNavMenu">
```

Comment:
It would be cool if this used [semantic tags](https://guide.freecodecamp.org/html/html5-semantic-elements/)

Reply:
I've tried to leave it as close to the original template as possible